### ISSUE TITLE
NVDA does not read quick fixes in eclipse 2019-06 for java developers sometimes

### ISSUE THREAD

**Issue by adriorjales** [nvaccess/nvda#9895]
You have arrived at a default issue template (bug report).
It is preferable to [choose the type of issue](https://github.com/nvaccess/nvda/issues/new/choose) to submit.
Hint: Swap to the preview tab to be able to click the links.
Direct links to the templates:
- [Bug report](https://github.com/nvaccess/nvda/issues/new?template=bug_report.md)
- [Feature request](https://github.com/nvaccess/nvda/issues/new?template=feature_request.md)

### Steps to reproduce:
Open Eclipse 2019-06
create a Java project
create a java package
create a java class
edit the class and make a issue in the code, like write ublic in stead of public
access the issue by pressing ctrl+. to select the issue
and press f2 to access quick fixes available.
Move up and down to explore fixes available and the issue will take place

### Actual behavior:
you will notice that nvda reads sometimes the fixes but no always. 

### Expected behavior:
NVDA must read the fixes available always

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2019.1.1

#### Windows version:
Windows 10 LTSC 2019

#### Name and version of other software in use when reproducing the issue:
Eclipse 2019-06 

#### Other information about your system:
16 GB core I5 ssd laptop

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by Adriani90** [comment]
@adriorjales are you still having this issue with NVDA 2024.2 and Eclipse 2024-06?

**Comment by Adriani90** [comment]
cc: @shadyar could you please also test whether this issue occurs with NVDA 2024.3 Beta or 2024.2 Stable?

**Comment by shadyar** [comment]
With NVDA 2024.2

**Comment by Adriani90** [comment]
Is it solved? Or can you still reproduce it?Von meinem iPhone gesendetAm 10.07.2024 um 16:00 schrieb Shadyar Khodayari ***@***.***>:﻿
With NVDA 2024.2

**Comment by shadyar** [comment]
The issue has been resolved.
